test/Concurrency/async_tasks.swift

```diff
@@ -0,0 +1,70 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-concurrency
+// REQUIRES: concurrency
+
+func someAsyncFunc() async -> String { "" }
+
+struct MyError: Error {}
+func someThrowingAsyncFunc() async throws -> String { throw MyError() }
+
+// non-async function with "weird shape" representing some API that has not adopted
+// swift concurrency yet, but we want to call it from an async function and make it
+// play nice with the rest of the system.
+func someManyCallbacksFunction(
```

Comment:
<clippy says>: Looks like a call from an async func test might be missing?

Reply:
Hm right, I might just delete this function; since there's no implementation here anyway it does not really differ much from tests already in `test_unsafeContinuations`.